gossip/src/crds_gossip_pull.rs

```diff
@@ -1148,6 +1148,7 @@ pub(crate) mod tests {
 
         let mut dest_crds = Crds::default();
         let new_id = solana_sdk::pubkey::new_rand();
+        let same_key = ContactInfo::new_localhost(&new_id, 0);
```

Comment:
any specific reason you moved this up a few lines?

Reply:
The test relies on this contact-info not being upserted as is commented out in line 1159.
With the new overrides logic, I need to initialize this contact-info first so that it has an earlier `outset`.